src/scanners/javascript.js

```diff
@@ -80,10 +81,15 @@ export default class JavaScriptScanner {
           var code = message.message;
 
           // Support 3rd party eslint rules that don't have our internal
-          // message structure.
+          // message structure and allow us to optionally overwrite
+          // their `message` and `description`
           if (_messages.hasOwnProperty(code)) {
             var shortDescription = _messages[code].message;
             var description = _messages[code].description;
+          } else if (ESLINT_OVERWRITE_MESSAGE.hasOwnProperty(message.ruleId)) {
+            var overwrites = ESLINT_OVERWRITE_MESSAGE[message.ruleId];
+            var shortDescription = overwrites.message || message.message;
+            var description = overwrites.description || message.description;
```

Comment:
I don't see the code getting set in my branch that uses this. Might want to add `code = overwrites.code || code`.

After discussion on IRC it looks like this is only used in the command line version so maybe not needed.